.github/workflows/linux.yml

```diff
@@ -0,0 +1,380 @@
+name: Linux (Ubuntu 20.04, Python 3.8)
+on:
+  workflow_dispatch:
+  pull_request:
+  merge_group:
+  push:
+    branches:
+      - master
+      - 'releases/**'
+
+concurrency:
+  # github.ref is not unique in post-commit
+  group: ${{ github.event_name == 'push' && github.run_id || github.ref }}-linux
+  cancel-in-progress: true
+
+env:
+  PYTHON_VERSION: '3.8'
+  OV_BRANCH: 'master'
+  OV_TARBALL: ''
+
+permissions: read-all # Required by https://github.com/ossf/scorecard/blob/e23b8ad91fd6a64a0a971ca4fc0a4d1650725615/docs/checks.md#token-permissions
+
+jobs:
+  openvino_download:
```

Comment:
let's define user friendly names with spaces, etc via `name: `

The comment is valid for all jobs in all pipelines.

Reply:
@ilya-lavrenov that is not a purpose of this PR, we could refactor the names separately, I'll add the names only for the new jobs in this PR.

Reply:
added names